geometry/geometry_config.py

```diff
@@ -601,8 +601,8 @@
 
         c.tauMudet.Xtot = c.tauMudet.XFe + 2 * c.tauMudet.LateralSupportX#now we need to include also supports.
         c.tauMudet.Ytot = c.tauMudet.YFe + c.tauMudet.UpperSupportY + c.tauMudet.LowerSupportY 
-        #c.tauMudet.Ztot = c.tauMudet.NRpc*c.tauMudet.ZRpc+c.tauMudet.NFethick*c.tauMudet.ZFethick + c.tauMudet.NFethin*c.tauMudet.ZFethin
-        c.tauMudet.Ztot = 3 * u.m #QUICK TEST FOR MAGNETIC SPECTROMETER
```

Comment:
Again, would be good to squash this with whichever commit introduced it.

Reply:
Ok, I did it with git rebase -i, then I put the squash to all the related commits